packages/apps-config/src/endpoints/testingRococo.ts

```diff
@@ -369,7 +369,7 @@ export function createRococo (t: TFunction): LinkOption[] {
         // Needs paraId >= 1000, below that is reserved for system parachains
         // Update paraId to new selection, remove this flag to enable
         isDisabled: true,
-        paraId: 188,
+        paraId: 1188,
```

Comment:
Same here, the 3 lines above can be removed.

Reply:
Done